Libraries/NetworkingLibraries.php

```diff
@@ -1617,7 +1648,7 @@ function GetTargetOfAttack($attackID)
           AddDecisionQueue("SETDQVAR", $mainPlayer, 0, 1);
           AddDecisionQueue("PASSPARAMETER", $mainPlayer, "Units", 1);
         }
-        AddDecisionQueue("SPECIFICCARD", $mainPlayer, "MAUL_TWI,$attackerIndex", 1);
+        AddDecisionQueue("SPECIFICCARD", $mainPlayer, "MAUL_TWI," . $attacker->Index(), 1);
```

Comment:
was this path tested? i would also look into ECL'ing Maul and using Maul Leader to use Overwhelm with Maul

Reply:
okay i did some tests. Maul looks good. i'm going to also test how this plays with my recent changes for when closing the combat chain state. we're getting issues on ping-first,shoot-first interactions

Reply:
Yes, I made sure to carefully review all instances of these changes and also tested all scenarios locally.